### ISSUE TITLE
error 

### ISSUE THREAD

**Issue by vikrambathala4** [georgesung/ssd_tensorflow_traffic_sign_detection#27]
I couldn't able to install Pickle on python 3 and also I am getting the following error. Please help me to resolve this.
![image](https://user-images.githubusercontent.com/29906210/35752100-ee6b3ff8-0817-11e8-9cdb-735afa2e74df.png)

**Comment by sudonto** [comment]
Find the aolution [here](https://stackoverflow.com/questions/41813665/tensorflow-slim-typeerror-expected-int32-got-list-containing-tensors-of-type).

**Comment by YashBansod** [comment]
well If you are using tensorflow >1.1.0 you will have to do the following changes:
- Line 60 of model.py `conf_loss = tf.nn.sparse_softmax_cross_entropy_with_logits(logits=logits, labels=y_true_conf)`

- Line 71 of model.py `loc_loss = tf.where(smooth_l1_condition, loc_loss_l2, loc_loss_l1)`

- Line 164 and 165 of model.py `final_pred_conf = tf.concat(preds_conf, 1)
    final_pred_loc = tf.concat(preds_loc, 1)`

and the *inference.py* should run

**Comment by mstmustisnt** [comment]
@YashBansod thank you, that helped!
